### PR TITLE
 Implemented a binary search Algorithm with a comparison against naiv…

### LINKED ISSUE #346: Binary search
an algorithm in python that finds and fetches data from a sorted list of items. 

please assign this to me, am from GSSOC'22

### PR DESCRIPTION
**Pull request by FatimahAdwan**
…e search timing

### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?
yes i have

### Description
A binary search Algorithm which is compared against a naive pattern, to show how better to use binary search especially when dealing with a large number of information

Fixes #346 

### Checklist

- [ ] I've read the contribution guidelines.
- [ ] I've checked the issue list before deciding what to submit.
- [ ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Pull request

### CODE REVIEW

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@FatimahAdwan 
 Do the requires changes !
1. Fix your `readme.md` file.
2. Add gif or photos about your project in `readme.md` file.

**Comment by FatimahAdwan** [comment]
> @FatimahAdwan Do the requires changes !
> 
> 1. Fix your `readme.md` file.
> 2. Add gif or photos about your project in `readme.md` file.

okay, am on it

**Comment by FatimahAdwan** [comment]
Hey kushal, it's me again, so I have been working on this since yesterday, atleast am able to push my updated readme for this project, pls help me check and do let me know

File: Desktop Application/Basic/Python/Binary search/REAME.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+ # **Binary search algorithm**
+---
+
+binary search algorithm built using python
+Binary search algorithm is an algorithm which helps us to sort an ordered list faster than just scanning through every single element to get the target which is very tiring and time consuming. 
+This Algorithm works on the principle of divide and conquer. 
+Binary search looks for a particular item by comparing the middle most item of the collection. 
+If a match occurs, then the index of item is returned. 
+If the middle item is greater than the item, then the item is searched in the sub-array to the left of the middle item.
```
Comment (Kushal997-das) [inline Desktop Application/Basic/Python/Binary search/REAME.md:9]: Delete this file .

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
You didn't change anything inside this folder so please delete this `[Desktop Application/Basic/Python/Binary search/REAME.md`

**Comment by FatimahAdwan** [comment]
> You didn't change anything inside this folder so please delete this `[Desktop Application/Basic/Python/Binary search/REAME.md`

i should delete it and put a new one?

**Review by Kushal997-das — COMMENTED** [review]
@FatimahAdwan You did changes in `[CONTRIBUTING.md]` file then you did you add other files as well please add only those files those are really changes by you .

@FatimahAdwan It's been so long still you are making mistake please make it correct or we gonna close this PR .

File: Desktop Application/Basic/Python/Binary search/Readme.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+This process continues on the sub-array as well until the size of the subarray reduces to zero.
+Along the way, this will be compared with Naive search and prove that Binary search is way faster than Naive search
+Naive patterns checks for all the characters of the main string to the pattern.
+
+# **Application**
+---
+* This algorithm is used to search element in a given sorted array with more efficiency.
+* It could also be used for few other additional operations for example, to find the smallest element in the array or to find the largest element in the array.
+
+![Output Of Binary Search Algorithm](/C:\Users\Fati\Desktop.png).
```
Comment (Kushal997-das) [inline Desktop Application/Basic/Python/Binary search/Readme.md:20]: Link is not working

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@FatimahAdwan Link is not working .

**Comment by FatimahAdwan** [comment]
okay, I think is done now.

**Comment by FatimahAdwan** [comment]
hi Kushal, am still waiting for you to check. thank you

**Review by Kushal997-das — COMMENTED** [review]
@FatimahAdwan This is a simple program not an any desktop application or project , we here at `project-guidance` only accept projects .

**Pull request closed without merging**